utils.go

```diff
@@ -148,8 +150,15 @@ func canCacheQuery(q []byte) bool {
 	if len(q) < len("SELECT") {
```

Comment:
The comment & check seems incorrect now.

Reply:
I fixed the check, thanks for that. About comment though I don't see why would it be incorrect, could you advise more on that? All tests are passing.